### PR TITLE
Bundle Inconsolata with Xi-Mac

### PR DESCRIPTION
**Pull request by nangtrongvuon**
This PR addresses issue #134.

This PR simply bundles the Inconsolata version taken from Google Fonts with Xi-Mac and also includes the Open Font License that comes with it.

### CODE REVIEW

File: XiEditor/AppDelegate.swift [hunk 1]
```diff
@@ -83,7 +83,7 @@ class AppDelegate: NSObject, NSApplicationDelegate, XiClient {
     var documentController: XiDocumentController!
 
     // This is set to 'InconsolataGo' in the user preferences; this value is a fallback.
```
Comment (cmyr) [inline XiEditor/AppDelegate.swift]: This comment is out of date. It's also not totally clear that we need a fallback font now, but we may be relying on it in certain places (like setting up the initial drawing metrics,  below)

Comment (cmyr) [reply XiEditor/AppDelegate.swift]: (to clarify: I think it's fine to leave the fallback)

Comment (nangtrongvuon) [reply XiEditor/AppDelegate.swift]: I made the comment change to clarify Inconsolata being bundled now.

**Comment by raphlinus** [comment]
Also looks good to me. I think the top-level README can be updated. First, the "users are encouraged to try Inconsolata" can be changed to: "The default font is Inconsolata, which is bundled with the app".

Second, I'd add a licensing section, right above `## Contributions`:

```
## License

This project is licensed under the Apache 2 [license](LICENSE). The bundled fonts are under a different license, the Open Font License. See the [fonts](fonts) directory for the fonts and associated license.
```

Thanks! Neat that `ATSApplicationFontsPath` makes this so easy, without having to write code to explicitly load fonts from the bundle.

**Comment by raphlinus** [comment]
@nangtrongvuon I'm curious exactly what versions these are. The ones from https://github.com/google/fonts/tree/master/ofl/inconsolata are now at version 2.001, while the ones in this PR seem to be version 2.000. I think there are some bugs in this one, and am looking at maybe updating to the latest version. I'm still digging into the question of which version is the authoritative best one...

**Comment by nangtrongvuon** [comment]
I took Inconsolata directly from Google Fonts site (picked Inconsolata, chose to download it only) about nearly 2 weeks ago. I tried to put in InconsolataGo instead but I couldn’t download it from your site (either dead link or inaccessible to me.)

**Pull request merged**